app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/constats.js

```diff
@@ -0,0 +1,2 @@
+export const JIRA_CLOUD = 'JIRA Cloud';
+export const AZURE_DEVOPS = 'Azure DevOps';
```

Comment:
Ideally the core UI shouldn't know any info about certain plugin, its name and description.